src/panels/developer-tools/state/developer-tools-state.ts

```diff
@@ -0,0 +1,682 @@
+          display: block;
+          padding: 16px;
+          padding: max(16px, env(safe-area-inset-top))
+            max(16px, env(safe-area-inset-right))
+            max(16px, env(safe-area-inset-bottom))
+            max(16px, env(safe-area-inset-left));
+        }
+
+        ha-textfield {
+          display: block;
+        }
+
+        .state-input {
+          margin-top: 16px;
+        }
+
+        ha-expansion-panel {
+          margin: 0 8px 16px;
+        }
+
+        .inputs {
+          width: 100%;
+          max-width: 800px;
+        }
+
+        .info {
+          padding: 0 16px;
+        }
+
+        .button-row {
+          display: flex;
+          margin-top: 8px;
+          align-items: center;
+        }
+
+        .table-wrapper {
+          width: 100%;
+          overflow: auto;
+        }
+
+        .entities th {
+          padding: 0 8px;
+          text-align: left;
+          font-size: var(
+            --paper-input-container-shared-input-style_-_font-size
+          );
+        }
+
+        .filters th {
+          padding: 0;
+        }
+
+        .filters search-input {
+          display: block;
+          --mdc-text-field-fill-color: transparent;
+        }
+        ha-tip {
+          display: flex;
+          padding: 8px 0;
+        }
+
+        th.attributes {
+          position: relative;
+        }
+
+        th.attributes ha-checkbox {
+          position: absolute;
+          bottom: -8px;
+        }
+
+        :host([rtl]) .entities th {
+          text-align: right;
+          direction: rtl;
+        }
+
+        :host([rtl]) .filters {
+          direction: rtl;
+        }
+
+        .entities tr {
+          vertical-align: top;
+          direction: ltr;
+        }
+
+        .entities tr:nth-child(odd) {
+          background-color: var(--table-row-background-color, #fff);
+        }
+
+        .entities tr:nth-child(even) {
+          background-color: var(--table-row-alternative-background-color, #eee);
+        }
+        .entities td {
+          padding: 4px;
+          min-width: 200px;
+          word-break: break-word;
+        }
+        .entities ha-svg-icon {
+          --mdc-icon-size: 20px;
+          padding: 4px;
+          cursor: pointer;
+          flex-shrink: 0;
+          margin-right: 8px;
+        }
+        .entities td:nth-child(1) {
+          min-width: 300px;
+          width: 30%;
+        }
+        .entities td:nth-child(3) {
+          white-space: pre-wrap;
+          word-break: break-word;
+        }
+
+        .entities a {
+          color: var(--primary-color);
+        }
+
+        .entities .id-name-container {
+          display: flex;
+          flex-direction: column;
+        }
+        .entities .id-name-row {
+          display: flex;
+          align-items: center;
+        }
+
+        :host([narrow]) .state-wrapper {
+          flex-direction: column;
+        }
+
+        :host([narrow]) .info {
+          padding: 0;
+        }
+
```

Comment:
Somehow, "narrow" isn't added anymore to the host element. Because of that, these css rules are not working anymore. I'm not sure ow to solve.

Reply:
See my suggestions